app/views/teams/show.html.erb

```diff
@@ -11,7 +11,7 @@
 <% end %>
 <% if admin? || current_user&.admin_of?(@team) %>
   <%= link_to 'Szerkesztés', edit_team_path(@team), class: 'ui button' %>
-  <%= link_to 'Törlés', team, method: :delete, data: { confirm: 'Biztosan törlöd?' }, :class => "ui button negative" %>
+  <%= link_to 'Törlés', team_delete_memberships_path(@team), method: :delete, data: { confirm: 'Biztosan törlöd?' }, :class => "ui button negative" %>
```

Comment:
Jelenleg az adott csapat adminja is tudja törölni, legyen úgy, hogy csak a site adminnak van hozzá jogosultsága.

Reply:
Inkább `team_path(@team)`-re mutasson a link.